airbyte-webapp/src/pages/connections/ConnectionJobHistory/ConnectionJobHistory.tsx

```diff
@@ -43,7 +43,7 @@ const getJobRunningOrPending = (jobs: JobWithAttemptsRead[]) => {
   });
 };
 
-export const ConnectionStatusPage: React.FC = () => {
+export const ConnectionJobHistory: React.FC = () => {
```

Comment:
It should be `ConnectionJobHistoryPage/ConnectionJobHistoryPage`. All pages components end with `Page`.